tests/trainer/test_sharded_checkpoint.py

```diff
@@ -105,99 +116,6 @@ def _compare_model_params_between_state_dicts(state_dict1, state_dict2):
                            state_dict2_model_tensor), f'Weight named {param_name} not the same between state_dicts'
 
 
-@pytest.mark.gpu
```

Comment:
could you please justify/explain deleting these tests?

Reply:
we talked offline. they aren't truly necessary to test functionality, but could be good for picking up on some weird torch warnings. will try to add back in